dart/dynamics/ReferentialSkeleton.hpp

```diff
@@ -460,6 +479,13 @@ class ReferentialSkeleton : public MetaSkeleton
   /// Name of this ReferentialSkeleton
   std::string mName;
 
+  /// Skeletons that this ReferentialSkeleton is referencing to.
```

Comment:
Same comment as above about this comment being ambiguous.

Reply:
Done.